packages/search-adapter/src/empathy/mappers/request/params/empathy-request-related-tags-query.mapper.ts

```diff
@@ -8,9 +9,29 @@ import { RequestMapper } from '../../../empathy-adapter.types';
  * @public
  */
 @injectable()
-export class EmpathyRequestRelatedTagsQueryMapper implements RequestMapper<QueryableRequest, string> {
-
+export class EmpathyRequestRelatedTagsQueryMapper
+  implements RequestMapper<QueryableRequest, string>
+{
   map({ relatedTags = [] }: QueryableRequest, query: string): string {
```

Comment:
I think we could even simplify this a little more. I'm not 100% sure again, so give me some time to see if I can break it, but here is the proposal:

```
  map({ relatedTags = [] }: QueryableRequest, query: string): string {
    return relatedTags.reduce(
      (partialQuery, relatedTag) =>
        relatedTag.query.startsWith(relatedTag.tag)
          ? `${ relatedTag.tag } ${ partialQuery }`
          : `${ partialQuery } ${ relatedTag.tag }`,
      query
    ).trim();
  }
```